guides/pages/how-to-modify-liveblocks-storage-from-the-server.mdx

```diff
@@ -134,16 +134,23 @@ export async function modifyStorage(
 
     // Make storage adjustments in a batch, so they all happen at once
     room.batch(() => {
-      storageChanges(root);
-
-      // Leave when storage has been synchronized
-      room.subscribe("storage-status", (status) => {
-        if (status === "synchronized") {
-          serverClient.leave(roomId);
-          resolve();
-        }
-      });
+      try {
+        storageChanges(root);
+      } catch (error) {
+        console.log("Problem modifying Liveblocks Storage from Node.js", error);
+      }
```

Comment:
Not sure if that console log should be in here. Ideally people would put that in the `storageChanges()` helper if they're interested in catching things there? Feel free to leave it off.